src/components/admin/app-breadcrumbs/Breadcrumbs.js

```diff
@@ -68,81 +68,91 @@ export default {
           displayName: 'menu.colorPickers'
         },
         {
-          path: 'notifications',
+          name: 'notifications',
           displayName: 'menu.notifications'
         },
         {
-          path: 'icons',
-          displayName: 'menu.icons'
+          name: 'icons',
+          displayName: 'menu.icons',
+          children: [
+            {
+              displayName: 'concrete',
+              name: '',
```

Comment:
And what this does?

Reply:
instead of using paths in breadcrumbs object i decided to use name of these routes, what helps to solve problem with incorrect paths

maybe "routeName" is more suitable here?

Reply:
Why name is "" then? Also ":name"?